test/e2e/service_bootstrap.py

```diff
@@ -55,6 +55,11 @@ def create_execution_role() -> str:
     iam.attach_role_policy(
         RoleName=role_name, PolicyArn="arn:aws:iam::aws:policy/AmazonS3FullAccess"
     )
+    # This policy accommodates use of the feature group controller.
+    iam.attach_role_policy(
```

Comment:
did you check what is present in this policy that is not in SagemakerFullAccess?